modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/CriteriaRow.es.js

```diff
@@ -347,7 +346,7 @@ class CriteriaRow extends Component {
 		const inputComponentsMap = {
 			[PROPERTY_TYPES.BOOLEAN]: BooleanInput,
 			[PROPERTY_TYPES.COLLECTION]: CollectionInput,
-			[PROPERTY_TYPES.DATE]: DateInput,
+			[PROPERTY_TYPES.DATE]: DateTimeInput,
```

Comment:
We should not import DateTime component if we are not using it.

Reply:
True, thanks